content/community/_index.html

```diff
@@ -13,8 +13,8 @@ <h1 class="title is-3">UQCS Community</h1>
                 In the past, we've run Games, Jackbox, Trivia and Movie Nights, BBQs, study sessions, end-of-semester
                 drinks, and workshops; we also frequently collaborate with other tech and related interests societies,
                 both in and out of UQ.</p>
-            <p>We have an extremely active Slack community consisting of thousands of members, comprised of current 
-                students, alumni, and other people in the Brisbane tech community.</p>
+            <p>We have an extremely active <a href="https://discord.uqcs.org/">Discord community</a> consisting of thousands
```

Comment:
We can no longer way we have thousands of students. The easiest fix might just be to change that to say hundreds

Reply:
Good point, have committed 7ab3b4e to fix this.

Reply:
Could also just not mention the count until it gets to the point of it being worth mentioning again

Reply:
Done 5a503c5